Dockerfile.linux-amd64

```diff
@@ -1,5 +1,5 @@
 ARG GO_VERSION
-FROM golang:$GO_VERSION-bullseye
+FROM golang:$GO_VERSION-bookworm
```

Comment:
I think it would be better to hardcode the version here and use renovate to create the update PRs for this. Then on master pushes we push a tag / create a release and in the release build make build is executed.

Reply:
Seems pretty hard for it to detect this templated tag.. I was worried it is going to take longer than it ever will doing it manually 🫠

Reply:
It's updated every month no? for every patch version of go. Can we do the same as in the gotify/server repo with a GO_VERISON file which is updated via renovate. THen we can use this file as value for the dockerfile argument.

Reply:
I think it's easier to me this just passively update corresponding new tags when we update the server repo to version. I doubt anything goes wrong bumping the go version here?

The workflow I was thinking is when renovate bump go version in the server repo detect if the corresponding build container is missing and if so trigger a CI here with that go version

Reply:
> I think it's easier to me this just passively update corresponding new tags when we update the server repo to version. I doubt anything goes wrong bumping the go version here? So we only need to intervene when Debian bumps version here.

I agree, renovate provides automatically merge PRs if the build succeeds.

> The workflow I was thinking is when renovate bump go version in the server repo detect if the corresponding build container is missing and if so trigger a CI here with that go version

How would you do this? Check in the build if the corresponding gotify/build image exists? This feels like an potential race condition where the gotify/build ci build is still running but the gotify/server check is retriggered. Yeah, you can guard against that but it feels much easier if gotify/server updates only if new gotify/build images exist. And the gotify/build images update only when there is a new golang image.

Reply:
If we set the CI for the update workflow to run sequentially there will be no conflict. And not sure how often the github hosted renovate runs but definitely the fastest it can is [every hour](https://docs.renovatebot.com/key-concepts/scheduling/#default-behavior)

Yeah I am debating these two options, if we are releasing a version for every single Go release your solution is cleaner if not I think it is better to just call build from the server repo.

Reply:
I can't find a way to ask renovate to 'check if the current tags include the newest version'..

Reply:
I'm not sure I understand what you mean with the last comment. I'd imaging it going like this:

```
gotify/build:
  with GO_VERSION (in gotify/build repo file, updated via renovate) if a new golang image releases

gotify/build:
  on master push (if image does not exist): make GO_VERSION=$(cat GO_VERSION) build push
                                          : and create tag with go version (after image is pushed)

gotify/server:
  GO_VERSION (updated via renovate) with gotify/build tags
  "regexManagers": [
    {
      "fileMatch": ["^GO_VERSION$"],
      "matchStrings": [
        "^(?<currentValue>\\d+\\.\\d+\\.\\d+)$"
      ],
      "datasourceTemplate": "github-tags",
      "depNameTemplate": "gotify/build",
      "versioningTemplate": "semver"
    }
  ]
```

Reply:
I'd create this file. It's more difficult to let renovate update the Dockerfile because the non amd64 Dockefiles reference the gotify/build images.